lib/core/src/server/manager/manager-webpack.config.js

```diff
@@ -65,6 +66,7 @@ export default ({ configDir, configType, entries, dll, outputDir, cache, babelOp
       extensions: ['.mjs', '.js', '.jsx', '.json'],
       modules: ['node_modules'].concat(raw.NODE_PATH || []),
       alias: {
+        ...themingPaths,
```

Comment:
nit: should we spread `uiPaths` first since it depends on theming?

Reply:
I think that it should go in order of leaves-to-root. Whatever spreads last will overwrite what came before, so if the idea is that the package closest to the root should get the last say in how the dependencies are resolved, then `@storybook/ui` should come last.